setup.py

```diff
@@ -17,7 +17,7 @@
 setup(
     name='perfkitbenchmarker',
     url='https://github.com/GoogleCloudPlatform/PerfKitBenchmarker',
-    version='0.6.0',
+    version='0.17.0',
```

Comment:
Since we don't really support installs via setup.py, it's not actually important. I may just change it to a fixed string ('dev'?).